src/charts/tooltip.js

```diff
@@ -595,6 +602,23 @@ define(function(require){
         };
 
         /**
+         * Exposes the ability to use a custom date format
+         * @param  {String} _x          Desired custom format
+         * @return {String | module}  Current format or module to chain calls
+         * @public
+         * @example tooltip.dateFormat(tooltip.axisTimeCombinations.CUSTOM);
+         * tooltip.customDateFormat('%H:%M %p')
+         */
+        exports.customDateFormat = function(_x) {
+            if (!arguments.length) {
+              return customDateFormat || '%m/%d/%Y @ %H:%M %p';
```

Comment:
I don't think we need this default here.
Nit: The indentation seems to be 2 spaces.

Reply:
Yeah, agreed. Fixed that and the spacing here: 4572fb93c437aff24095140383102233494cb06a.